### PR TITLE
[Just Eat] Tabletless info

### PR DESCRIPTION
**Pull request by antmonnier**

### CODE REVIEW

**Comment by antmonnier** [comment]
@janaina-wittner Can you confirm it is accurate?

File: content/apps/just-eat-flyt/en/overview.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ Just Eat Flyt Bridge allows you to connect to HubRise the following platforms:
 
 ---
 
+---
+
+**IMPORTANT NOTE:** Tabletless integration is supported in certain markets, such as the UK market. To check if your market is supported, contact us.
```
Comment (janaina-wittner) [inline content/apps/just-eat-flyt/en/overview.md:42]: I would rather say : 

**IMPORTANT NOTE:** The OderPad can be switched of in the UK market. The Just Eat Flyt integration does not support tabletless integrations on any other market.

No need to give hope when there is none.
Even in IE, where Flyt only is used, the orderpad cannot be switched off. 

Just Eat refers to OrderPad, not tablet but we can leave both for SEO.

File: content/apps/just-eat-flyt/fr/presentation-generale.md [hunk 2]
```diff
@@ -19,6 +19,12 @@ app_info:
 
 ---
 
+---
+
+**REMARQUE IMPORTANTE :** L'intégration sans tablette est possible pour certains marchés, comme le marché anglais. Pour savoir si votre marché est couvert, contactez-nous.
```
Comment (janaina-wittner) [inline content/apps/just-eat-flyt/fr/presentation-generale.md:24]: Same here.

File: content/apps/just-eat-takeaway/en/overview.md [hunk 3]
```diff
@@ -40,6 +40,12 @@ Just Eat Takeaway Bridge allows you to connect the following platforms to HubRis
 
 ---
 
+---
+
+**IMPORTANT NOTE:** Tablets are required to receive orders in HubRise with the Just Eat Takeaway integration. However, Just Eat Flyt integration allows to switch off tablets, in some markets. Contact us to check if you are eligible to the latter integration.
```
Comment (janaina-wittner) [inline content/apps/just-eat-takeaway/en/overview.md:45]: I would not give any hope on this: 
**IMPORTANT NOTE:** Tablets are required to receive orders in HubRise with the Just Eat Takeaway integration. 

In the UK, the Just Eat Takeaway integration is never used and it's the only market we can switch off the tablet.

File: content/apps/just-eat-takeaway/fr/presentation-generale.md [hunk 4]
```diff
@@ -22,6 +22,12 @@ app_info:
 
 ---
 
+---
+
+**REMARQUE IMPORTANTE:** Les tablettes de commande doivent rester allumées pour recevoir les commandes dans HubRise avec l'intégration Just Eat Takeaway. Cependant, l'intégration Just Eat Flyt permet d'éteindre les tablettes, pour certains marchés. Contactez-nous pour vérifier votre éligibilité à cette intégration.
```
Comment (janaina-wittner) [inline content/apps/just-eat-takeaway/fr/presentation-generale.md:27]: Same here

**Review by janaina-wittner — COMMENTED** [review]
@antmonnier, see my comments.

**Comment by antmonnier** [comment]
Implemented @janaina-wittner feedback

**Pull request merged**